<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order_management</class>
 <widget class="QMainWindow" name="order_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>1005</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order Management</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/logo.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lineEdit_vender">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vender</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="vender">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="book_name">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Book Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_book_name">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter valid book name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="author_list">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Author List</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_author1">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Author 1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_author2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Author 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit_author4">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
            </property>
            <property name="placeholderText">
             <string>Author 4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_author3">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Author 3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="publication">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Publication</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_publication">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter Publication</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="release_year">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Realeas Year</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_realeas_year">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="quantity">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_quantity">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter Book Quantity</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="price">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_price">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter book price</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="edition">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Edition</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboBox_book_edition">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="total">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_total">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding:4px;</string>
     </property>
     <property name="text">
      <string>Generate Quotation</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
